File: app/src/main/java/com/project/scratchstudio/kith_andoid/network/model/board/BoardsResponse.java
package com.project.scratchstudio.kith_andoid.network.model.board;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;

public class BoardsResponse {

    @SerializedName("boards")
    private ArrayList<Board> boards;

    public ArrayList<Board> getBoards() {
        return boards;
    }
}

File: app/src/main/java/com/project/scratchstudio/kith_andoid/network/model/board/BoardsResponse.kt
package com.project.scratchstudio.kith_andoid.network.model.board

import com.google.gson.annotations.SerializedName
import com.project.scratchstudio.kith_andoid.network.model.BaseResponse

import java.util.ArrayList

class BoardsResponse : BaseResponse() {

    @SerializedName("boards")
    lateinit var boards: ArrayList<Board>
}
